src/EnergyPlus/RuntimeLanguageProcessor.cc

```diff
@@ -2151,24 +2171,43 @@ namespace RuntimeLanguageProcessor {
 			for ( OperandNum = 1; OperandNum <= ErlExpression( ExpressionNum ).NumOperands; ++OperandNum ) {
 				Operand( OperandNum ) = ErlExpression( ExpressionNum ).Operand( OperandNum );
 				if ( Operand( OperandNum ).Type == ValueExpression ) {
-					Operand( OperandNum ) = EvaluateExpression( Operand( OperandNum ).Expression ); //recursive call
+					Operand( OperandNum ) = EvaluateExpression( Operand( OperandNum ).Expression, seriousErrorFound ); //recursive call
 				} else if ( Operand( OperandNum ).Type == ValueVariable ) {
-					Operand( OperandNum ) = ErlVariable( Operand( OperandNum ).Variable ).Value;
+					if ( ErlVariable( Operand( OperandNum ).Variable ).Value.initialized ) { // check that value has been initialized
+						Operand( OperandNum ) = ErlVariable( Operand( OperandNum ).Variable ).Value;
+					} else { // value has never been set
+						ReturnValue.Type = ValueError;
+						ReturnValue.Error = "EvaluateExpression: Variable = '" + ErlVariable( Operand( OperandNum ).Variable ).Name + "' used in expression has not been initialized!" ;
+						if ( ! DoingSizing && ! KickOffSimulation && ! EMSManager::FinishProcessingUserInput ) {
+
+							//check if this is an arg in CurveValue,
+							if ( ErlExpression( ExpressionNum ).Operator != FuncCurveValue ) { // padding the argument list for CurveValue is too common to fatal on.  only reported to EDD
```

Comment:
As discussed above, the guidance given to users for CurveValue was to pad the extra arguments with dummy variables.  It seems every one would be likel to fatal out, so that is why the exclusion.